Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -994,13 +996,32 @@ def test():
 
 
 ''' EXECUTION CODE '''
-client = resilient.get_client({
-    'email': USERNAME,
-    'password': PASSWORD,
-    'host': SERVER,
-    'cafile': 'true' if USE_SSL else 'false',
-    'org': ORG_NAME
-})
+
+
+def get_client():
+    if USERNAME and PASSWORD:
+        resilient_client = resilient.get_client({
```

Comment:
id prefer we init the client once, so lets build the args dict, assign it in the conditions and the pass it with the `**` operator